test/integration/default/controls/_mapdata_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Keep only first 2 digits from Ubuntu finger
+mapdata_file = "_mapdata/#{system.platform[:finger].split('.').first}.yaml"
+
+# Load the mapdata from profile https://docs.chef.io/inspec/profiles/#profile-files
+mapdata_dump = inspec.profile.file(mapdata_file)
```

Comment:
I think this should be in within the control block - the reason being that it actually executes an operation.
When picking out individual controls to run (for example, in `kitchen.yml`) random other operations shouldn't be run too.
```suggestion
# Load the mapdata from profile https://docs.chef.io/inspec/profiles/#profile-files
mapdata_dump = inspec.profile.file(mapdata_file)
```

Reply:
Thanks, @dafyddj -- I'll make this change as well as the others discussed above.

Reply:
Done; also moved the rest of the assignments into the `control` block to keep things tidy.